README.md

```diff
@@ -76,9 +86,9 @@ cd docs && hugo serve
 If you would like to see how the latest website looks based on the documentation in the Iceberg repository, you can copy docs to this repository by:
 ```shell
 rm -rf docs/content/docs
-rm -rf landing-page/content/common
-cp -r <path to iceberg repo>/docs/versioned docs/content/docs
-cp -r <path to iceberg repo>/docs/common landing-page/content/common
+rm -rf landing-page/content/common/format
+cp -r <path to iceberg repo>/docs docs/content/docs
+cp -r <path to iceberg repo>/format landing-page/content/common/format
```

Comment:
I went ahead and flattened the `landing-page/content/common` directory since those urls are not tied to the folder structure. Updated this to say `..../format/* -> ..../common/`.